vcf/filter.go

```diff
@@ -40,6 +41,10 @@ func Filter(v Vcf, chrom string, minPos int, maxPos int, ref string, alt []strin
 	if onlyPolarizableAncestors && !IsPolarizable(v) {
 		return false
 	}
+	//raven's note: added a check for id
+	if !FilterId(v, id) {
```

Comment:
I might consider the style of the above "if" statements above where you first check to see if a filter exists and then apply the filter (example: if id != "" && IsIdMatch(v, id))
Right now "FilterId" has the check (id != "") hidden in it, where it might not be obvious from the documentation that the blank string is a wildcard that will match everything.

Reply:
I just made this change in my new branch. Thank you!